java.lexer/src/org/netbeans/lib/java/lexer/JavaLexer.java

```diff
@@ -1051,7 +1051,12 @@ else if ('0' <= c && c <= '9') { // float literal
 
                                     if (AFTER_VAR_TOKENS.contains(next.id())) {
                                         do {
+                                            if (next.partType() != PartType.COMPLETE) {
```

Comment:
It's rather unexpected usage of PartType, but should work. I would still prefer to just check for `null` after `nextToken()` call, as Javadoc comments can have some embeddings injected which may eventually produce start-middle-end tokens. I am not 100% sure how the token system works in that case.

@jlahoda ?

Reply:
Had added null check as @sdedic suggested in last commit